appserver/persistence/cmp/support-sqlstore/pom.xml

```diff
@@ -117,7 +117,7 @@
         </resources>
         <plugins>
             <plugin>
-                <groupId>org.codehaus.mojo</groupId>
+                <groupId>org.omnifaces</groupId>
```

Comment:
Is this completely new plugin or is it based on codehause's dead or is it based no this one?
https://www.antlr.org/api/maven-plugin/latest/plugin-info.html

Reply:
It's the codehouse plug-in, updated to all the latest versions of everything, and with the correct settings to not use the security manager to prevent system.exit, but instead throw an exception.